Makefile

```diff
@@ -49,6 +49,17 @@ control-plane-dev-docker-multi-arch: check-remote-dev-image-env ## Build consul-
        --push \
        -f $(CURDIR)/control-plane/Dockerfile $(CURDIR)/control-plane
 
+control-plane-fips-dev-docker: ## Build consul-k8s-control-plane FIPS dev Docker image.
```

Comment:
Maybe for completeness, could you add the fips version of `control-plane-dev-docker-multi-arch`? It is the only target I can use on M1s 😢

Reply:
I initially attempted to do this, but cross-compilation for the correct C tooling proved hard enough that I just gave up and switched to developing on a linux/amd64 instance.

see https://github.com/golang/go/issues/54695 for more info